processing/src/main/java/org/apache/druid/query/DruidProcessingConfig.java

```diff
@@ -40,9 +41,14 @@
   private AtomicReference<Integer> computedBufferSizeBytes = new AtomicReference<>();
 
   @Config({"druid.computation.buffer.size", "${base_path}.buffer.sizeBytes"})
+  public String bufferSizeConfigured()
+  {
+    return "-1";
+  }
+
   public int intermediateComputeSizeBytesConfigured()
   {
-    return DEFAULT_PROCESSING_BUFFER_SIZE_BYTES;
+    return (int) Bytes.parse(bufferSizeConfigured(), DEFAULT_PROCESSING_BUFFER_SIZE_BYTES);
```

Comment:
I guess we can call `Bytes.parse()` without default value since `bufferSizeConfigured()` has default?

Reply:
Hmm, this method can be called several times during processing a query. Why not returning `Bytes` in `bufferSizeConfigured()` instead of `String`?

Reply:
Yes, it should return an instance of `Bytes`. I didn't understand how `@Config` works at first, so `String` was used.